pkg/costmodel/router.go

```diff
@@ -887,6 +888,10 @@ func Initialize(additionalConfigWatchers ...ConfigWatchers) {
 		Name: "kubecost_cluster_management_cost",
 		Help: "kubecost_cluster_management_cost Hourly cost paid as a cluster management fee.",
 	}, []string{"provisioner_name"})
+	LBCostRecorder := prometheus.NewGaugeVec(prometheus.GaugeOpts{ // don't know if necessary to differentiate ELB vs. ALB cost
+		Name: "load_balancer_cost",
```

Comment:
Nice PR!

We didn't prefix any of our original metrics with `kubecost_` but we've started doing with recording rules, etc. What do you guys think we should do here? @itscalvinwang @mbolt35 @AjayTripathy @nikovacevic